CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 - Added support for `Index.to_numpy`.
 - Added support for `DataFrame.align` and `Series.align` for `axis=0`.
 - Added support for `size` in `GroupBy.aggregate`, `DataFrame.aggregate`, and `Series.aggregate`.
+- Added partial support for `snowflake.snowpark.functions.window`
```

Comment:
move it up to 1.25.0

Reply:
should we call it partial support? lets documents its capabilities to function description and not word it as partial support.